tests/bots/test_mellow.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import variables
+from blockchain.deposit_strategy.base_deposit_strategy import MellowDepositStrategy
+from bots.depositor import DepositorBot
+
+from tests.conftest import COUNCIL_ADDRESS_1, COUNCIL_ADDRESS_2, COUNCIL_PK_1, COUNCIL_PK_2
+from tests.utils.protocol_utils import get_deposit_message
+
+
+# this fixture must be executed before others
+@pytest.fixture(autouse=True)
+def setup_environment():
+    variables.MELLOW_CONTRACT_ADDRESS = '0x078b1C03d14652bfeeDFadf7985fdf2D8a2e8108'
+
+
+# 20529904 - is the block with mellow tx
+# 20529995 - is the block with the fallback to regular tx
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    'web3_provider_integration,module_id',
+    [[20529904, 2], [20529995, 2]],
+    indirect=['web3_provider_integration'],
+)
+def test_depositor_bot_mellow_deposits(
+    web3_provider_integration,
+    web3_lido_integration,
+    deposit_transaction_sender_integration,
+    mellow_deposit_strategy_integration,
+    base_deposit_strategy_integration,
+    gas_price_calculator_integration,
+    module_id,
+    add_accounts_to_guardian,
+):
+    # Define the whitelist of deposit modules
+    variables.DEPOSIT_MODULES_WHITELIST = [1, 2]
```

Comment:
Maybe set here 
`variables.DEPOSIT_MODULES_WHITELIST = [2]`

Reply:
I intentionally set it to `[1, 2]` because the bots variable will contain not only mellow module